src/posts/test-structuring.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: 'Test Structuring in C#'
+tags:
+  - post
+  - programming
+---
+
+Im changing my opinion on test structuring in C#. I've always thought that structuring tests using a class hierarchy made sense because it provided a lot of benefits:

```

Comment:
```suggestion
I'm changing my opinion on test structuring in C#. I've always thought that structuring tests using a class hierarchy made sense because it provided a lot of benefits:

```